packages/react-table/src/components/Table/EditableSelectInputCell.tsx

```diff
@@ -18,8 +18,7 @@ export interface IEditableSelectInputCell extends Omit<React.HTMLProps<HTMLEleme
     event: React.MouseEvent | React.ChangeEvent,
     newValue: any | any[],
     rowIndex: number,
-    cellIndex: number,
-    isPlaceholder?: boolean
```

Comment:
If we're removing this we should update the "Editable rows" example in the deprecated Table example file to remove the param and usage from the onSelect handler (~line 690)